content/en/docs/concepts/scheduling-eviction/pod-scheduling-readiness.md

```diff
@@ -91,7 +91,7 @@ scheduling. You can use `scheduler_pending_pods{queue="gated"}` to check the met
 
 ## Mutable Pod Scheduling Directives
 
-{{< feature-state for_k8s_version="v1.27" state="beta" >}}
+{{< feature-state for_k8s_version="v1.30" state="stable" >}}
```

Comment:
I would remove this completely since line 9 covers the whole feature, and this is part of it .

Reply:
sg, let me remove it in this section.